test/integration/users.test.js

```diff
@@ -469,7 +469,7 @@ describe("Users", function () {
           return done();
         });
     });
-    it("Should return all users for empty value of search param", function (done) {
+    it("Should return 400 for empty value of search param", function (done) {
```

Comment:
this looks like a hot fix and it is not a part of discord API PR.

Please improve the test and can it be raised as a separate pr, we can get that merged before our discord API pr

Reply:
Resolved